crates/core/tedge_mapper/src/aws/mapper.rs

```diff
@@ -72,6 +72,7 @@ impl TEdgeComponent for AwsMapper {
             mqtt_schema,
             tedge_config.aws.mapper.timestamp_format,
             prefix.clone(),
+            tedge_config.aws.mapper.mqtt.max_payload_size,
```

Comment:
Related to my comment above, for example, here we can add the size check for `aws.mapper.mqtt.max_payload_size`

Reply:
Implemented the validation at the `tedge_config` level itself, so that the user gets the feedback much earlier.